changelog.tmpl

```diff
@@ -0,0 +1,40 @@
+{{- if index .NotesByType "breaking-change" -}}
```

Comment:
Minor thing, could we move this file and `note.tmpl` into a subdirectory? Could they also be put into `.changelog/` ? Or maybe somewhere  else?

Reply:
Yeah absolutely. Happy to put wherever we want that to live.